docs/content/gameobject/reading-and-writing-component-data.md

```diff
@@ -73,6 +73,9 @@ public class ReadHealthBehaviour : MonoBehaviour
 2. Send a component update to specify the new component values that your component should be updated to using `Writer.Send(TComponentUpdate update)`.
 </br>(`ISpatialComponentUpdate` types are generated under `<namespace of schema component>.<component name>.Update`.) 
 
+**Known Issue Warning:** 
```

Comment:
Known issue warning  <- note captitalisation

(Note we use newspaper titles, which are the same as sentences but with no full stops.)

Reply:
Fixed.